db/migrate/20171005192813_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[5.1]
+  def change
+    create_table :comments do |t|
+      t.text :body
+      t.references :user, foreign_key: true, null: false
```

Comment:
I am a little bit confused. What does `user` relation mean? and what does `author_id` mean?

Reply:
@jizak user_id stands for user that receives the comment, author_id - commenter.id. What would be the better naming here?

Reply:
I think it makes sense to add foreign key and relation to `author` also.